recipe/SPQR_TBB.patch

```diff
@@ -0,0 +1,11 @@
+--- SPQR/Lib/Makefile	2016-05-04 16:47:24.000000000 -0400
++++ SPQR/Lib/Makefile.new	2016-05-31 15:04:27.075922172 -0400
+@@ -13,7 +13,7 @@
+ include ../../SuiteSparse_config/SuiteSparse_config.mk
+ 
+ # SPQR depends on CHOLMOD, AMD, COLAMD, LAPACK, the BLAS and SuiteSparse_config
+-LDLIBS += -lamd -lcolamd -lcholmod -lsuitesparseconfig $(LAPACK) $(BLAS)
++LDLIBS += -lamd -lcolamd -lcholmod -lsuitesparseconfig $(LAPACK) $(BLAS) $(TBB)
```

Comment:
I didn't need it on OS X, but for some reason when I tried locally on my linux machine I was getting errors related to TBB.  I can try removing it here and see what happens.